packages/app-project/helpers/getProjectSlugFromURL/getProjectSlugFromURL.js

```diff
@@ -0,0 +1,10 @@
+function getProjectSlugFromURL (url) {
+  const relativeUrl = (url.startsWith('http'))
+    ? url.replace(/^(?:\/\/|[^/]+)*\//, '')
+    : url
+
+  const parts = relativeUrl.split('/').filter(Boolean)
```

Comment:
1. A test spec for `.filter(Boolean)`? 
2. At this point, there _could_ be a falsy value, but only if the `url` arg isn't actually a URL...

Reply:
I'm all for safety checks. I think having a test is good to make sure it's doing what we think it is and making it clear for other devs what it's there for.

Reply:
[PFE uses filter(Boolean)](https://github.com/zooniverse/Panoptes-Front-End/search?q=filter%28Boolean%29&unscoped_q=filter%28Boolean%29) in a few places too. I don't think we should be writing tests for `Array.filter` but we should not be assuming anything about the value of the URL arg either, if it's come from the browser.

Reply:
Ok, I see your point too. I guess I'm mostly just reacting to my unfamiliarity of using `.filter(Boolean)` this way, but it was also an easy Google search for an explanation so, maybe a spec is overkill.

Reply:
Maybe add some tests for URLs that don't match the expected pattern? All the current tests are for URLs that look more-or-less valid.

Reply:
This only ever gets called with a relative URL, so I'm going to find out what's valid for `owner` and `slug` and write a regex to parse that, rather than a URL on its own